#time
time timeMsgs

##### mission
duration durationMission

##### subtask
duration durationSubmission
int32 idSubmission
bool loopSubmission


##### task
duration durationTask
int32 idTask
int32 taskType 

#task state
int32 taskState
int32 WAITING_BRAIN=1
int32 TASK_RUNNING=2
